m2cgen/interpreters/code_generator.py

```diff
@@ -119,3 +130,6 @@ class CLikeCodeGenerator(BaseCodeGenerator):
     tpl_else_statement = CodeTemplate("} else {")
     tpl_block_termination = CodeTemplate("}")
     tpl_var_assignment = CodeTemplate("${var_name} = ${value};")
+
+    scalar_variable_type = "double"
```

Comment:
`variable` is a bad name, since we use this for method return type as well. How about `output`?

Reply:
Done